src/stirling/source_connectors/socket_tracer/kafka_trace_bpf_test.cc

```diff
@@ -28,7 +28,8 @@
 #include "src/common/testing/test_utils/container_runner.h"
 #include "src/common/testing/testing.h"
 #include "src/stirling/source_connectors/socket_tracer/protocols/kafka/common/types.h"
-#include "src/stirling/source_connectors/socket_tracer/testing/container_images.h"
+#include "src/stirling/source_connectors/socket_tracer/testing/container_images/kafka_container.h"
+#include "src/stirling/source_connectors/socket_tracer/testing/container_images/zookeeper_container.h"
```

Comment:
This test is [disabled](https://github.com/pixie-io/pixie/blob/f4f4ec2d16cc66306ac6d35c0bf6a34ab528aab3/src/stirling/source_connectors/socket_tracer/BUILD.bazel#L416) under qemu. Can you force it to run locally to verify that even though it fails the containers run for the test?

Reply:
Ran with the `no_qemu` tag removed and the containers spin up properly even though some of the assertions fail.